### DISCUSSION TITLE
Barrel roll problems. Is it planned to be like this?

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#20676] [Q&A]
Barrel roll mod can place objects almost out of screen. Example in video.
(I have vertical scaling 70%)
It's CS 5 map. I can click on this barely displayed circles but it's very strange when they are out of playfield (and out of screen without scaling).
https://user-images.githubusercontent.com/55319814/194793282-5d5f2bd5-502a-4d58-bf1c-9dbf569a38ef.mp4

You can say "No one mapping like this" but here is example. Road of resistance by Monstrata:
![osu_2022-10-10_06-14-28](https://user-images.githubusercontent.com/55319814/194794645-db30595c-ca2f-4043-a6d0-35edf7dbb436.jpg)

**Comment by peppy** [comment]
Your video won't load, but I'm guessing it's due to mappers putting objects outside the playfield as per the screenshot.

**Reply by okolenmi** [reply]
Example on screen works good (playfield has an extra space out of mapping field) but Barrel Roll mod can destroy it.  Video works for me (click on link, I don't know how to embed it).

**Reply by mk56-spn** [reply]
it should embed if you separate it from the previous text with an empty line
